src/cap-description.adoc

```diff
@@ -37,6 +91,56 @@ All locations in registers or memory able to hold a capability are CLEN+1 bits
 wide including the tag bit. Those locations are referred as being _CLEN-bit_ or
 _capability_ wide in this specification.
 
+==== Bounds
+
+A capability represents an interval portion of the address space.  This is
+expressed using a *base*, which is an unsigned integer less than 2^CXLEN^, and
+a *length*, which is an unsigned integer less than or equal to 2^CXLEN^.  The
+sum of the base and the length is also called the *top*, and will be no more
+than 2^XLEN^ for a well-formed capability.  A capability with base *B* and
+length *L* authorizes access to all bytes of the address space whose address
+*A* satisfies *B* <= *A* < *B* + *L*, as further defined by the sealing and
+architectural permission fields.  In other words, a capability represents a
+half-open interval.
+
+Capability operations always treat addresses as unsigned.  A capability's valid
+region can span the gap between 2^CXLEN-1^-1 and 2^CXLEN-1^, but cannot wrap
+around from 2^CXLEN^-1 to 0.
+
+Subset and equality operations treat capability bounds as intervals, not as
+sets.  Two capabilities with length 0 and different bases are not equal and not
+subsets of each other.  A capability of length 0 can be constructed only inside
+an existing capability for an interval which surrounds the base.
+
+The bounds and address of a capability are subject to alignment and
```

Comment:
I think it is very important to not representability, but I think the more detailed part with the table should be part of the bounds format section instead?